doc/content/concepts/devicetemplates/assigning-euis.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Assigning EUIs"
+description: ""
+weight: 6
+---
+
+You can use the `device templates assign-euis` command to assign the `JoinEUI` and `DevEUI` to device templates. The `DevEUI` is assigned from the given start number. This personalizes the device templates.
+
+You can use the command in two different ways:
+
+1. Pipe a number of device templates to assign EUIs to, for example the result of [Mapping Templates](../mapping) or [Converting Templates](../converting)
+2. Pipe a generic device template, for example the result of [Creating Templates](../creating), and pass the `--count` flag to generate a number of personalized device templates
+
+Once the device templates are personalized with EUIs, you can set additional fields (see [Creating Templates](../creating)) or execute templates (see [Executing Templates](../executing)).
+
+>Note: to make your device compliant with most LoRaWAN networks, use the `JoinEUI` of the LoRaWAN Join Server where the devices will be registered and use a `DevEUI` from an IEEE issued MAC address block that is assigned to your company. If you intend to operate your own LoRaWAN Join Server, use a `JoinEUI` from your IEEE assigned MAC address block as LoRa Alliance requires that for DNS lookup. [Learn how to apply for a IEEE issued MAC address block](https://standards.ieee.org/products-services/index.html).
+
+## Example
+
+This examples illustrates creating a device profile template, assigning 5 `DevEUI`s and creating them in your TTN Stack application.
+
+First, create a mapping file with a device profile in `profile.json`:
+
+```bash
+$ ttn-lw-cli device template create \
+  --frequency-plan-id EU_863_870 \
+  --lorawan-version 1.0.3 \
+  --lorawan-phy-version 1.0.3-a \
+  --supports-join > profile.json
+```
+
+Second, assign the EUIs:
+
+```bash
+$ cat profile.json \
+  | ttn-lw-cli device template assign-euis 70b3d57ed0000000 70b3d57ed0000001 --count 5 > devices.json
+```
+
+<details><summary>Show output</summary>
```

Comment:
Nice. Does this also work in Hugo? If so, then we should use it more. If not, we should make it work.

Reply:
Yeah that works. We should take care of template support.

Edge doesn't support it but it will come most probably when they switch to Chromium. See compatibility table here; https://www.w3schools.com/tags/tag_details.asp

Reply:
Then it should be defined as a shorcode so we can easily use it in other sections of the doc.
https://gohugo.io/content-management/shortcodes/